mailpoet/assets/js/src/global.d.ts

```diff
@@ -102,7 +102,6 @@ interface Window {
   mailpoet_wp_week_starts_on: number;
   mailpoet_subscribers_counts_cache_created_at: string;
   mailpoet_shortcode_links: string[];
-  mailpoet_settings: any;
   mailpoet_tracking_config: string;
```

Comment:
This is not a string but an object. Maybe we can update it to the correct type `{level:"basic"|"partial" etc...}`?

Reply:
Fixed in https://github.com/mailpoet/mailpoet/pull/4312/commits/4312b9aa9e71356f75b4da5249615eef2d512a31.